bookmarks/vanilla/src/modules/Bookmark.ts

```diff
@@ -156,90 +164,81 @@ export class Bookmark {
      * @param title string
      * @param link string
      * @param session Session
-     * @returns IBookmark[]
+     * @returns IBookmark
```

Comment:
What if the referenced item does not exist? I think we have to throw an exception here and also mention it in the function header.

Reply:
it will be considered in https://github.com/solid-contrib/data-modules/issues/32